queries.sql

```diff
@@ -10,3 +10,89 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS null;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT save_one;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO save_one;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+SELECT COUNT (id) FROM animals;
+SELECT COUNT (escape_attempts) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' 
+GROUP BY species;
+
+
+SELECT name, full_name
+FROM animals
+JOIN owners
+ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name as animal, species.name as species
+FROM animals
+JOIN species
+ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+SELECT full_name as owner, name as animal
+FROM animals
+FULL JOIN owners
+ON animals.owner_id = owners.id;
+
+SELECT species.name as species, Count(animals.name) as counted
+FROM animals
+JOIN species
+ON animals.species_id = species.id
+GROUP BY species.name;
```

Comment:
- [ ] Great job on your progress, and kudos for addressing the previous reviewer's feedback. I understand you encountered issues with pgAdmin 4 and switched to using the shell. However, as per the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Remember%20to%20add%20the%20screenshot%20with%20the%20results%20of%20your%20queries%20to%20your%20Pull%20Request%20description.), please consider adding the screenshot to the PR description, even if you had to use a different method to capture it. This will help ensure you meet all project requirements and provide a clear overview of your work. Keep up the good work! 👍📷💼

Reply:
I added the screenshot, it is there!!!

Reply:
@Kingno3l , as per the project requirement, you have to add it to the PR description.
![image](https://github.com/Kingno3l/Vet-Clinic/assets/55840999/c8e2ac22-6588-4610-b204-4f4aad5105c9)